src/main/webapp/js/components/entities/entity-table.jsx

```diff
@@ -45,6 +45,9 @@ class EntityTable extends Component {
 
 const mapStateToProps = (state, props) => ({
   entity: getEntityArrayByName(state, props),
+  fieldConfig: getVisibleFields(state, {
```

Comment:
EntityTable is getting the field config as a parameter from ViewEntity, so you don't need to get it from redux